### ISSUE TITLE
Text editing isn't accessible to screen readers

### ISSUE THREAD

**Issue by ghost** [GoogleChromeLabs/text-app#106]
Text editing isn't accessible to screen readers.

When the user appears to be editing the document, focus is actually inside an empty textarea that's positioned where the cursor should be.
